### PR TITLE
FSRS-5.md

### PR DESCRIPTION
**Pull request by Expertium**
This is a draft. Don't merge this yet.

### CODE REVIEW

**Comment by L-M-Sherlock** [comment]
I have updated the table in main branch.

File: README.md [hunk 1]
```diff
@@ -20,38 +20,38 @@ Note: TimeSeriesSplit will remove the first split from evaluation. This is becau
...
 - Transformer: a type of neural network that has gained popularity in recent years due to its superior performance in natural language processing. ChatGPT is based on this architecture.
-- SM-2: one of the early algorithms used by SuperMemo, the first spaced repetition software. It was developed more than 30 years ago, and it's still popular today. [Anki's default algorithm is based on SM-2](https://faqs.ankiweb.net/what-spaced-repetition-algorithm.html), [Mnemosyne](https://mnemosyne-proj.org/principles.php) also uses it.
+- SM-2: one of the early algorithms used by SuperMemo, the first spaced repetition software. It was developed more than 30 years ago, and it's still popular today. [Anki's default algorithm is based on SM-2](https://faqs.ankiweb.net/what-spaced-repetition-algorithm.html), [Mnemosyne](https://mnemosyne-proj.org/principles.php) also uses it. Theis algorithm does not predict the probability of recall natively; therefore, for the sake of the benchmark, the output was modified based on some assumptions about the forgetting curve.
```
Comment (L-M-Sherlock) [inline README.md:42]: This?

File: README.md [hunk 2]
```diff
@@ -20,38 +20,38 @@ Note: TimeSeriesSplit will remove the first split from evaluation. This is becau
...
 
-For more details about the FSRS algorithm, read this wiki page: [The Algorithm](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm).
+For further information regarding the FSRS algorithm, please refer to the following wiki page: [The Algorithm](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm).
 
 ## Result
 
 Total number of users: 19,990.
 
-Total number of reviews for evaluation: 707,964,360. Same-day reviews are excluded; only one review per day (the first chronologically) is used by each algorithm. This is why the number of reviews is significantly lower than the 1.5 billion mentioned earlier. Plus, an outlier filter is also used.
+Total number of reviews for evaluation: 707,964,360. Same-day reviews are excluded; only one review per day (the first chronologically) is used by each algorithm, except for FSRS-5. This is why the number of reviews is significantly lower than the 1.7 billion figure mentioned earlier. Plus, an outlier filter is also used.
```
Comment (L-M-Sherlock) [inline README.md:52]: Manual rescheduling revlogs are also removed.

File: README.md (hunk 1 above)
Comment (Expertium) [reply README.md:42]: My bad, that's a typo

File: README.md (hunk 2 above)
Comment (Expertium) [reply README.md:52]: I'll add that

**Review by L-M-Sherlock — APPROVED** [review]
LGTM

**Pull request merged**